plugin.rb

```diff
@@ -93,7 +93,11 @@ class Engine < ::Rails::Engine
           count: reaction.reaction_users_count,
         }
 
-        if DiscourseReactions::Reaction.reactions_counting_as_like.include?(reaction.reaction_value)
+        # NOTE: It does not matter if the reaction is currently an enabled one,
+        # we need to handle historical data here too so we don't see double-ups in the UI.
+        if !DiscourseReactions::Reaction.reactions_excluded_from_like.include?(
```

Comment:
Seeing `DiscourseReactions::Reaction.reactions_excluded_from_like.include?` in multiple places. Maybe should be a method on `DiscourseReactions::Reaction`?

Reply:
Hmm I think it's fine, it's only 2 places and just an array check